code/app.py

```diff
@@ -331,20 +342,9 @@ def conversation_custom():
         chat_history = []
         for i, k in enumerate(user_assistant_messages):
             if i % 2 == 0:
-                chat_history.append(
-                    (
-                        user_assistant_messages[i]["content"],
-                        user_assistant_messages[i + 1]["content"],
-                    )
-                )
-        from utilities.helpers.ConfigHelper import ConfigHelper
-
-        messages = message_orchestrator.handle_message(
-            user_message=user_message,
-            chat_history=chat_history,
-            conversation_id=conversation_id,
-            orchestrator=ConfigHelper.get_active_config_or_default().orchestrator,
-        )
+                chat_history.append((user_assistant_messages[i]['content'],user_assistant_messages[i+1]['content']))
```

Comment:
```suggestion
                chat_history.append((user_assistant_messages[i]["content"], user_assistant_messages[i+1]["content"]))
```

I'm sorry, I get so distracted by style inconsistencies. I think this repo doesn't enforce style linting yet, but perhaps you can manually lint your additions to match the existing style in terms of spaces and quotes.
Or just ignore my style nits.

Reply:
black and flake8 have recently been introduced as a git-precommit. Looks like it has not been installed (or not running from the devcontainer). I think this will greatly help a lot of the style comments you are mentioning

Reply:
Ah ross just said that the codebase has black now, yay!